infra/base-images/base-clang/checkout_build_install_llvm.sh

```diff
@@ -17,7 +17,7 @@
 
 NPROC=16  # See issue #4270. The compiler crashes on GCB instance with 32 vCPUs.
 
-LLVM_DEP_PACKAGES="build-essential make cmake ninja-build git python2.7 g++-multilib"
+LLVM_DEP_PACKAGES="build-essential make cmake ninja-build git python2.7 g++-multilib binutils-dev"
```

Comment:
Is the gold version good enough from ubuntu 16.04 image ?

Reply:
yes, that is no problem. (for afl++ is it even unimportant as I link directly with lld)